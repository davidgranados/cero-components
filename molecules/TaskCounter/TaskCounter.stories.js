import { TaskCounter, styles, options } from '.'

import { getTemplate, getOptionsArgTypes } from '../../helpers/storybook'

const Template = getTemplate(TaskCounter, styles)

export default {
  title: 'Molecules/TaskCounter',
  component: TaskCounter,
  args: {
    children: 'Em Sm√©agol body.',
    current: 2,
  },
  argTypes: {
    types: getOptionsArgTypes(options.types),
  },
}

export const Default = Template.bind({})
export const WithTotal = Template.bind({})
WithTotal.args = { total: 10 }
