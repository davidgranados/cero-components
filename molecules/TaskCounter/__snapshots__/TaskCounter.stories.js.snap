// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Molecules/TaskCounter Default 1`] = `
<div
  style={
    Object {
      "alignContent": "flex-start",
      "display": "flex",
      "flexDirection": "column",
      "flexWrap": "wrap",
      "gap": "10px 30px",
      "height": "100%",
      "justifyContent": "flex-start",
      "maxHeight": "auto",
    }
  }
>
  <div
    className="task-counter"
  >
    <div
      className="container"
    >
      <div
        className="paragraph color-base size-md weight-medium"
      >
        Em Sméagol body.
      </div>
      <div
        className="paragraph color-base size-md weight-medium is-inline"
      >
        2
      </div>
    </div>
    <div
      className="spacer horizontal"
      style={
        Object {
          "height": 5,
        }
      }
    />
    <div
      className="divider"
    />
  </div>
</div>
`;

exports[`Storyshots Molecules/TaskCounter With Total 1`] = `
<div
  style={
    Object {
      "alignContent": "flex-start",
      "display": "flex",
      "flexDirection": "column",
      "flexWrap": "wrap",
      "gap": "10px 30px",
      "height": "100%",
      "justifyContent": "flex-start",
      "maxHeight": "auto",
    }
  }
>
  <div
    className="task-counter"
  >
    <div
      className="container"
    >
      <div
        className="paragraph color-base size-md weight-medium"
      >
        Em Sméagol body.
      </div>
      <div
        className="paragraph color-base size-md weight-medium is-inline"
      >
        2/10
      </div>
    </div>
    <div
      className="spacer horizontal"
      style={
        Object {
          "height": 5,
        }
      }
    />
    <div
      className="divider"
    />
  </div>
</div>
`;
